chainer/functions/noise/simplified_dropconnect.py

```diff
@@ -136,6 +142,9 @@ def simplified_dropconnect(x, W, b=None, ratio=.5, train=True, mask=None):
             Otherwise, The mask must be ``(n, M, N)`` shaped array.
```

Comment:
In current implementation, `mask` shape input by user is not checked. So user can input any shape. Should we check `mask` shape? And expected shape should be changed by `use_batchwise_mask`?